src/a-d/Alligator/Alligator.StreamHub.cs

```diff
@@ -1,25 +1,11 @@
 namespace Skender.Stock.Indicators;
 
-// WILLIAMS ALLIGATOR (STREAM HUB)
-
 /// <summary>
-/// Represents a stream hub for calculating the Alligator indicator.
+/// Streaming hub for Alligator indicator.
```

Comment:
This doesn't appear to follow the convention we discussed before, for StreamHub class name.  I'm glad to see you found one that you missed previously.  We don't put "indicator." at the end of these -- that's the convention for series style.

Think harder about how these are written.